arch/power/Makefile.in

```diff
@@ -0,0 +1,41 @@
+# Makefile for zlib-ng
+# Copyright (C) 1995-2013 Jean-loup Gailly, Mark Adler
```

Comment:
This file doesn't exist in stock zlib, so the copyright line is completely incorrect.

Reply:
Agreed. But does this also apply to `arch/x86/Makefile.in`, `arch/arm/Makefile.in` and `arch/s390/Makefile.in`? I don't think they are part of stock zlib either.

Reply:
It should apply to all new files, not just under arch subdirectories... I haven't checked all the files before they were committed as @Dead2 does the actual merging in.

Reply:
Ok, I just updated it to something similar as I used in `slide_hash_power8.c`